Makefile

```diff
@@ -217,6 +217,23 @@ release-image-name: ## Prints the image name of the release image
 release-image-tag: ## Prints the image tag of the release image
 	@echo $(IMAGE_TAG)
 
+##############################
+## End-to-end (E2E) Testing ##
+##############################
+
+e2e-setup: ## Setup end-to-end tests
+	@cd e2e && npm install
+	@cd e2e && npx playwright install --with-deps
+
+e2e-setup-ci: ## Install dependencies and Playwright browsers
+	cd e2e && npx playwright install --with-deps
+
+e2e-test: ## Run end-to-end tests
+    # make e2e-test APP_NAME=app BASE_URL=http://localhost:3000
```

Comment:
looks like a leftover line. delete?

Reply:
It's an inline example of how to use it - is it better in docs?

Reply:
yeah probably better in docs